logoapp/upload.php

```diff
@@ -1,10 +1,76 @@
 <?php
-echo '<pre>';
-echo $_POST["image"];
+echo "処理中です…<br>";
 
-if(move_uploaded_file($_POST["image"], "upload/test.jpg")){
+require('set_db.php');
+$id = "NuLl";
+
+//echo '<pre>';
+//print_r($_POST);
+
+$image = $_POST["image"];
+//echo "image: " .$image;
+//echo '<br>';
+$data = $_POST["data"];
+//echo "data: " .$data;
+//echo '<br><br>';
+$type = $_POST["type"];
+//echo "type: ".$type;
+//echo '<br><br>';
+
+if(strcmp($type,"dot") == 0){//dotかlineかを確認
```

Comment:
__[code_sniffer]__
* Expected 1 space after IF keyword; 0 found
* No space found after comma in function call
* Expected 1 space after closing parenthesis; found 0

<!-- Comment by [SideCI](https://sideci.com) -->